### ISSUE TITLE
Growl: Inaccessible for screen readers

### ISSUE THREAD

**Issue by MarkusChur** [primefaces/primefaces#5151]
# Summary
When using screen readers growl messages are not read out at all. It seems like attributes are not set correctly so that assistive technology is not able to interpret what's going on.
There are two use cases: POST requests that update the whole page and AJAX requests that update the growl. I provide a PR that makes it a bit better but due to browser and screen reader incompatibilities it's currently not possible to support every combination.

## 1) Environment
Does not work with 7.0.6
Does not work with the showcase
https://www.primefaces.org/showcase/ui/message/growl.xhtml

## 2) Expected behavior
Growl message is read out.

## 3) Actual behavior
Growl message is not read out.

## 4) Steps to reproduce
Use the showcase and trigger a growl message. Listen to the screen reader output.

## 5) AJAX current showcase.

| Browser | NVDA | JAWS |
| --- | --- | --- |
| Chrome | no | no |
| IE11 | no | no |
| Firefox | no | no |
| Edge | no | no |

## 6) AJAX fixed version

| Browser | NVDA | JAWS |
| --- | --- | --- |
| Chrome | yes | yes |
| IE11 | yes | no |
| Firefox | yes | yes |
| Edge | no | no |

## 7) POST fixed version

| Browser | NVDA | JAWS |
| --- | --- | --- |
| Chrome | yes | yes |
| IE11 | yes | no |
| Firefox | yes | no |
| Edge | no | no |

**Comment by melloware** [comment]
Thanks for the PR and the testing!
